packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -21,6 +21,7 @@ import {
 } from '../../Utilities';
 import { compareDates, compareDatePart } from '../../utilities/dateMath/DateMath';
 import * as stylesImport from './DatePicker.scss';
+import { FocusTrapZone } from 'office-ui-fabric-react/lib/FocusTrapZone';
```

Comment:
This import should be a relative path.

Reply:
fixed now, thanks for helping figure this out David